api/views.py

```diff
@@ -94,23 +94,28 @@ class DocumentViewSet(viewsets.ReadOnlyModelViewSet):
         )
 
 class SpellFilter(django_filters.FilterSet):
+    # TODO the filter for level_int was lost in the modelfield rename, replace this.
```

Comment:
I see a `level_int` filter. Is this TODO resolved? If not, what needs to be done?

Reply:
It works automatically! Django knows that level_int is an alias of spell_level.